src/controller/editingcontroller.js

```diff
@@ -23,6 +23,7 @@ import {
 } from '../conversion/model-selection-to-view-converters';
 
 import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import '../../theme/theme.scss';
```

Comment:
I was thinking that this might be connected with a way how at the end we will build editor with proper theme. I am not sure if this is defined yet but let's wait for @Reinmar response.